codewars20.py

```diff
@@ -0,0 +1,21 @@
+# https://www.codewars.com/kata/520b9d2ad5c005041100000f/train/python
+
+def pig_it(text):
+    lst_text = list(text)
+    chars = "abcdefghijklmnopqrstuvwxyz"
```

Comment:
The solution is good and works fine. Just another small tip that you might find useful in the future, there's a library with the letters of the alphabet. You can read about it and its usages here: https://stackoverflow.com/a/16060908

Reply:
This is very helpful, thank you! And huge thanks for your encouragement!